benchmarks/k8s/base/codespeed.yaml

```diff
@@ -0,0 +1,151 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: agent-k8s-codespeed
+spec:
+  template:
+    spec:
+      shareProcessNamespace: true
```

Comment:
Would be good to add a comment here on why we need it - because capture script needs access to /proc/<pid> for the agent process...